app/javascript/mastodon/features/compose/components/poll_form.js

```diff
@@ -155,7 +155,7 @@ class PollForm extends ImmutablePureComponent {
         <div className='poll__footer'>
           <button disabled={options.size >= 4} className='button button-secondary' onClick={this.handleAddOption}><Icon id='plus' /> <FormattedMessage {...messages.add_option} /></button>
 
-          <select value={expiresIn} onChange={this.handleSelectDuration}>
+          <select value={expiresIn} onBlur={this.handleSelectDuration}>
```

Comment:
This would introduce a bug (see #12596)

Reply:
I've reverted this, and added a line to configure the linter to ignore this.